rlImGui/rlImGui.cs

```diff
@@ -658,6 +687,22 @@ public static void Image(Texture2D image)
             ImGui.Image(new IntPtr(image.Id), new Vector2(image.Width, image.Height));
         }
 
+        /// <summary>
+        /// Draw a texture as an image in an ImGui Context
+        /// Uses the current ImGui Cursor position and the full texture size.
+        /// </summary>
+        /// <param name="image">The raylib texture to draw</param>
+        /// <param name="tintColor">The color to tint the image with</param>
+        public static void Image(Texture2D image, Color tintColor)
+        {
+            ImGui.Image(
+                new IntPtr(image.Id),
+                new Vector2(image.Width, image.Height),
+                Vector2.Zero, Vector2.One,
+                new Vector4(tintColor.R / 255f, tintColor.G / 255f, tintColor.B / 255f, tintColor.A / 255f)
+            );
+        }
+
```

Comment:
I prefer to keep rlImGui-cs feature compatible with the C++ version.
I'm not against having a tint version, but it should not duplicate code like this.
The tint should be an optional argument on the existing functions, or the non-tint versions should call the tint versions with a default tint. This feature would also need to be added to the C++ version for parity.

Reply:
Alright, I have fixed the code duplication problem, and I'll begin working on a PR to the C++ version with the same features.